docs/relnotes.rst

```diff
@@ -34,9 +34,40 @@ Version 1.0.0
 =============
 *2021 December 10*
 
-**JWST Improvements**:
+For JWST, this release includes updates to WebbPSF just prior to the launch. For Roman, it includes updates to use the Cycle 9 optical model results.
+
+**James Webb Space Telescope OTE model improvements**:
+* Updates in sign conventions for representing WFE, for strict consistency with the JWST WSS and other tools. Much of this was implemented by upstream changes in ``poppy``;
+  see `this page in the poppy docs <https://poppy-optics.readthedocs.io/en/latest/sign_conventions_for_coordinates_and_phase.html>`_ for details.  (:pr:`397`, :pr:`419` by :user:`mperrin`, :pr:`418` by :user:`Skyhawk172`)
+* Add models of OTE field dependence from the nominal OTE design and as-built optics (:pr:`389` by :user:`grbrady`, :pr:`505` by :user:`mperrin) and from
+  any misalignment of the secondary mirror, such as would be measured and corrected in MIMF (:pr:`392` by :user:`Skyhawk172`). These
+  additions were enabled by more consistent use of JWST Linear Optical Model framework behind the scenes (:pr:`378` by :user:`mperrin).
+* Add an option to use a lookup table of field dependent OPDs from Ball's ITM tool (for JWST team internal use in
+  pre-launch wavefront team practices and rehearsals). (:pr:`425` by :user:`Skyhawk172`, :pr:`474` by :user:`mperrin`))
+* Update the JWST OTE Linear Model to allow more flexible pupil sampling, in particular using higher sampling to reduce Fourier aliasing in certain FGS calculations (:pr:`440` by :user:`kjbrooks`)
+* New capability for visualizing the JWST optical budget terms as represented in WebbPSF. See :doc:`jwst_optical_budgets`.
+
+
+**James Webb Space Telescope instrument model improvements**:
+* MIRI: Minor updates to pixel scale and rotation (:pr:`456` by :user:`mperrin`),
+  an improved model of the MIRI imager detector cross artifact (:pr:`417` by :user:`mperrin`)
+  and correctly label MIRI's P750L prism for the LRS mode as a prism, not a grating (:pr:`477` by :user:`mperrin` and `:user:`skendrew`)
+* MIRI: Add capability for shifting MIRI coronagraph masks, consistent with NIRCam sim capabilities (:pr:`428` by :user:`JarronL`)
+* NIRCam: Higher fidelity model of NIRCam weak lenses, including field dependence, non-linear interactions between lenses,
+  and as-built measured performances. (:pr:`496` by :user:`mperrin`, using results of calibration work by Randal Telfer)
+* All SIs: Substantial performance improvements speeding up the calculation of optical distortion (:pr:`429` by :user:`jarronL`)
+
+**Nancy Grace Roman Space Telescope model improvements**:
+* Use of Cycle 9 optical and integrated modeling results.
```

Comment:
Are there PRs we can reference for this line?

Reply:
It would be #466, though there should probably be more detail. I wrote a summary in that pull request's original comment and can can work on adapting it.

Reply:
Yes thanks @ojustino - I put that line in as a placeholder to get started and it definitely should be improved!

Reply:
Justin please feel free to also just push additional commits into this branch for that text